lib/plugins/aws/lib/naming.js

```diff
@@ -400,10 +400,13 @@ module.exports = {
     }${this.provider.getStage()}`;
   },
   getAlbTargetGroupName(functionName, albId, multiValueHeaders) {
-    return crypto
+    const hash = crypto
       .createHash('md5')
       .update(this.getAlbTargetGroupNameTagValue(functionName, albId, multiValueHeaders))
       .digest('hex');
+
+    const albTargetGroupName = this.provider.getAlbTargetGroupPrefix() + hash;
+    return albTargetGroupName.length > 32 ? albTargetGroupName.substr(0, 32) : albTargetGroupName;
```

Comment:
`substr` is deprecated, let's use `slice`

Reply:
Switched to slice